app/assets/javascripts/Components/Result/result.jsx

```diff
@@ -364,7 +364,7 @@ class Result extends React.Component {
     });
   };
 
-  destroyMark = (criterion_type, criterion_id) => {
+  destroyMark = (criterion_id) => {
```

Comment:
Did you look for where this method is called? It's very suspicious to change the signature of a function but not change anywhere else in the code.